Headers/IOKit/bluetooth/IOBluetoothHCIController.h

```diff
@@ -0,0 +1,400 @@
+/*
+ *  Released under "The GNU General Public License (GPL-2.0)"
+ *
+ *  Copyright (c) 2021 cjiang. All rights reserved.
+ *
+ *  This program is free software; you can redistribute it and/or modify it
+ *  under the terms of the GNU General Public License as published by the
+ *  Free Software Foundation; either version 2 of the License, or (at your
+ *  option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful, but
+ *  WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ *  or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ *  for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ *
+ */
+
+#ifndef _IOKIT_IOBLUETOOTHHCICONTROLLER_H
+#define _IOKIT_IOBLUETOOTHHCICONTROLLER_H
+
+#include <IOKit/bluetooth/Bluetooth.h>
+#include <IOKit/usb/IOUSBHostDevice.h>
+#include <os/log.h>
+
+#include <Availability.h>
+
+#ifndef __MAC_OS_X_VERSION_MIN_REQUIRED
+#error "Missing macOS target version"
+#endif
+
+class IOWorkQueue;
+class IODisplayWrangler;
+class IOBluetoothACPIMethods;
+class IOBluetoothPacketLogger;
+class IOBluetoothHostController;
+class IOBluetoothHostControllerTransport;
+class IOBluetoothHostControllerUSBTransport;
+class IOBluetoothHostControllerUARTTransport;
+
+struct BluetoothHardwareListType
+{
+    IOBluetoothHostControllerTransport  * mBluetoothTransport; //0
+    IOBluetoothHostController           * mBluetoothHostController; //8
+    //16
```

Comment:
Are these offsets? If so, are placeholders (17, 18) covered by the IOBluetoothHostController struct? Otherwise the type may be different from the kext one. Perhaps add static assertions?